kakao/src/main/kotlin/com/agoda/kakao/Actions.kt

```diff
@@ -790,3 +791,34 @@ interface WebActions {
         web.withElement(ref).perform(action)
     }
 }
+
+/**
+ * Provides action for TabLayout
+ */
+interface SpinnerActions : BaseActions {
+    /**
+     * Selects a spinner item at given position
+     *
+     * @param position Int
+     */
+    fun selectItem(position: Int) {
+        click()
+        // Clicks on the first matched item from the list.
+        Espresso.onData(Matchers.allOf(Matchers.`is`(Matchers.instanceOf(String::class.java))))
```

Comment:
I'm not sure that this is the right way to go. What if you have another data of the `String` class that is present on the screen? How it will behave?

Reply:
I can understand I just focus on the default string. If there's a customize view put in, it can't find the item.
I'll check again when we use a customized view.